specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2023-06-01-preview/SourceControls.json

```diff
@@ -163,6 +163,10 @@
               "$ref": "#/definitions/SourceControl"
             }
           },
+          "404": {
+            "description": "Not Found",
+            "x-ms-error-response": true
+          },
```

Comment:
This response code is not required in swagger: [https://github.com/Azure/azure-openapi-validator/blob/main/docs/no-error-code-responses.md](https://github.com/Azure/azure-openapi-validator/blob/main/docs/no-error-code-responses.md).